agent/consul/state/acl_oss.go

```diff
@@ -0,0 +1,501 @@
+// +build !consulent
+
+package state
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/consul/agent/structs"
+	memdb "github.com/hashicorp/go-memdb"
+)
+
+///////////////////////////////////////////////////////////////////////////////
+/////                          ACL Table Schemas                          /////
```

Comment:
These should be identical to the schemas removed from the other file right?

Reply:
yes